app/src/main/res/values/dimens.xml

```diff
@@ -1,13 +1,31 @@
 <resources>
```

Comment:
please refrain from adding too many different values and follow material guidelines https://material.io/archive/guidelines/layout/metrics-keylines.html#metrics-keylines-keylines-spacing
the layout doesn't have to be 100% aligned with the design